packages/vite/src/utils.ts

```diff
@@ -1,26 +1,24 @@
 import { createHash } from 'node:crypto'
 import path from 'node:path'
-import { statSync } from 'node:fs'
 import type { ImageConfig } from 'imagetools-core'
 
 export const createBasePath = (base?: string) => {
   return (base?.replace(/\/$/, '') || '') + '/@imagetools/'
 }
 
-export async function generateImageID(url: URL, config: ImageConfig, imageBuffer: Buffer) {
+export function generateImageID(url: URL, config: ImageConfig, imageHash: string) {
```

Comment:
it occurs to me that we don't really need the `url` anymore now that we're no longer using the mtime. the image hash should be unique enough. I think removing it would simplify this function a fair amount

Reply:
agreed. nice catch